engines/bops_consultees/app/views/bops_consultees/consultee_responses/_form.html.erb

```diff
@@ -22,9 +22,9 @@
         multiple: true %>
 
   <%= form.govuk_text_field :email,
-        label: {text: "Officer submitting comment"},
+        label: {text: "Your email address"},
         hint: {text: "Ensure your email address is correct to receive any further correspondence or consultations."},
-        value: @consultee.email_address %>
+        value: @last_consultee_response&.email.presence || @consultee.email_address %>
```

Comment:
If `@last_consultee_response` is only going to be used for this, then maybe the nil check and fallback could move into the controller to reduce the amount of logic in the view.